### DISCUSSION TITLE
Can the search bar be moved at the bottom?

### DISCUSSION THREAD

**Discussion by skamsie** [ibhagwan/fzf-lua#1470] [Q&A] [answered]
I would like to have the search bar at the bottom of the window similar to the telescope default, something like the screenshot below? Is this possible with fzf-lua?

![Screenshot 2024-10-02 at 18 27 52](https://github.com/user-attachments/assets/e15908b1-1493-4b3c-bb54-3b9d6d77e8d9)

**Comment by ibhagwan** [comment]
Use `--layout` in fzf_opts, read man fzf to see which layout you want.

**Reply by skamsie** [reply, marked as answer]
Thanks @ibhagwan , I achieved what I wanted with the following:

```lua
files = { fzf_opts = { ['--layout'] = 'default' } }
```
